### ISSUE TITLE
Databinding does not works correctly in 'header' property of paper-datatable-column element 

### ISSUE THREAD

**Issue by jesuslopezlugo** [David-Mulder/paper-datatable#115]
Hi David!

I found a bug with the data binding of 'header' property in paper-datatable-column. I have this:

```
 <paper-datatable-card>
 <paper-datatable>
 <paper-datatable-column header="[[keyTitle]]" property="key" type="String" sortable>
....
...
...
 </paper-datatable>
 </paper-datatable-card>
```

I changed the value of keyTitle, but nothing happen, even if I try to use the reset function of paper-datatable

**Comment by RoXuS** [comment]
Yes this PR fix the issue https://github.com/David-Mulder/paper-datatable/pull/78. 

The author does not seem to be active, so I did a new similar project where I am reactive. 

https://github.com/RoXuS/paper-datatable-api/
